styles.css

```diff
@@ -0,0 +1,176 @@
+body {
+  font-family: "Josefin Sans";
+  background-color: #fffafb;
+}
```

Comment:
I suggest you put the universal selector above the body selector. This will make resetting CSS easier 😊 .